src/sparseml/pytorch/sparsification/quantization/quantize.py

```diff
@@ -214,6 +221,72 @@ def add_input_activation_quant_wrappers(module: Module) -> Module:
     return module
 
 
+def add_output_activation_observers(module: Module):
+    """
+    implementation of torch.quantization add_observers_ that only adds observers
+    according to attached quantization_scheme properties. the existing implementation
+    (1.9+) includes its own logic for propagating including overriding set qconfigs
+    for certain activations without the ability to disable this behavior
+
+    :param module: module to add output activation observers to
+    """
+    # adapted from torch/ao/quantization/quantize.py::_add_observer_
```

Comment:
```suggestion
    # adapted from torch/ao/quantization/quantize.py::_add_observer_
    # source: https://github.com/pytorch/pytorch/blob/v1.13.0/torch/ao/quantization/quantize.py#L135
```

Reply:
Or i guess the 1.9 version since it seems like it was that version https://github.com/pytorch/pytorch/blob/880c811aeb6937a44d5cc700ab3d802f95538aa9/torch/quantization/quantize.py#L90

Reply:
thanks, added with a flake8 override for line length

Reply:
it's adapted across multiple versions for compatibility